.docker/Dockerfile

```diff
@@ -118,10 +118,14 @@ RUN sudo wget https://packages.osrfoundation.org/gazebo.gpg -O /usr/share/keyrin
 RUN sudo apt-get -q update \
     && sudo apt-get -q -y upgrade \
     && sudo apt-get -q install --no-install-recommends -y \
+    python3-pexpect \
     python3-wxgtk4.0 \
+    python3-future \
     rapidjson-dev \
     xterm \
-    libgz-sim7-dev \
+    libgz-sim?-dev \
+    gz-transport?? \
```

Comment:
It's just a regex;  rather than dealing with "is it `libgz-sim7-dev` or `libgz-sim8-dev`"

Could be dealt with in other, more elegant (e.g. more restrictive regexp) ways.

Reply:
Got it, thanks!